fmriprep/cli/run.py

```diff
@@ -717,18 +712,32 @@ def parse_spaces(opts):
 The argument "MNI152NLin6Asym:res-2" has been automatically added to the list of output spaces \
 (option ``--output-spaces``).""", file=stderr)
 
-    if opts.cifti_output and 'MNI152NLin2009cAsym' not in output_spaces:
-        if 'MNI152NLin2009cAsym' not in output_spaces:
-            output_spaces['MNI152NLin2009cAsym'] = {'res': 2}
-            print("""Option ``--cifti-output`` requires functional images to be resampled to \
-``MNI152NLin2009cAsym`` space. Such template identifier has been automatically added to the \
-list of output spaces (option "--output-space").""", file=stderr)
-        if not [s for s in output_spaces if s in ('fsaverage5', 'fsaverage6')]:
-            output_spaces['fsaverage5'] = {}
-            print("""Option ``--cifti-output`` requires functional images to be resampled to \
-``fsaverage`` space. The argument ``fsaverage:den-10k`` (a.k.a ``fsaverage5``) has been \
-automatically added to the list of output spaces (option ``--output-space``).""", file=stderr)
+    if opts.cifti_output:
+        grayords = {'91k': '32k', '170k': '59k'}  # CIFTI total grayords to surface densities
+        output_spaces['fsLR'] = {'den': grayords[opts.cifti_output]}
```

Comment:
Do you really want to force GIFTI outputs if you're having CIFTI outputs? If they're not explicitly asking for it, it's a lot of duplicated output.

Reply:
that's how we are currently handling CIFTIs, i.e. fsaverage5/6 surfaces are always produced atm.

I hope to get this functionality in, and then address the inherent problem after (#1892)

Reply:
Okay.